docs/known_issues/index.md

```diff
@@ -15,18 +15,17 @@ The following compiler issues are known and we are working on it. Please do not
 
 ### Release
 
-resolc `0.1.0-dev.7`
+resolc `0.1.0-dev.10`
 
 ### Missing features
 
 - [Libraries with public functions are not supported](https://github.com/paritytech/revive/issues/91)
 - [Automatic import resolution is not supported](https://github.com/paritytech/revive/issues/98)
 - The emulated EVM linear contract memory is limited to 64kb in size. Will be fixed with support for metered dynamic memory.
-- [Gas limits for contract calls are ignored](https://github.com/paritytech/revive/issues/117)
 - [EIP-4844 opcodes are not supported](https://github.com/paritytech/revive/issues/64)
 - IPFS metadata hashes are not supported
 - [Compiled contract artifacts can exceed the pallet static memory limit and fail to deploy](https://github.com/paritytech/revive/issues/96).
-- [Compiling contracts requiring `solc` version earlier than `0.8.0` requires using the EVM legacy assembly pipeline. This is in a highly experimental state.](https://github.com/paritytech/revive/issues/89)
+- Compiling contracts requiring `solc` version earlier than `0.8.0` are not supported.
```

Comment:
Should mention that maybe we will not be adding support never. Now it reads as if we are planning to add this feature.

Reply:
ACK, briefly changed it to reflect it better at the right place.